doc/build-windows-msvc.md

```diff
@@ -0,0 +1,85 @@
+# Windows / MSVC Build Guide
+
+This guide describes how to build bitcoind, command-line utilities, and GUI on Windows using Micsrosoft Visual Studio.
+
+For cross-compiling options, please see [`build-windows-mingw.md`](./build-windows-mingw.md).
+
+## Preparation
+
+### 1. Visual Studio
+
+This guide relies on using CMake and vcpkg package manager provided with the Visual Studio installation.
+Here are requirements for the Visual Studio installation:
+1. Minimum required version: Visual Studio 2022 version 17.6.
+2. Installed components:
+- The "Desktop development with C++" workload.
+
+The commands in this guide should be executed in "Developer PowerShell for VS 2022" or "Developer Command Prompt for VS 2022".
+The former is assumed hereinafter.
+
+### 2. Git
+
+Download and install [Git for Windows](https://git-scm.com/download/win). Once installed, Git is available from PowerShell or the Command Prompt.
+
+### 3. Clone Bitcoin Repository
+
+Clone the Bitcoin Core repository to a directory. All build scripts and commands will run from this directory.
+```
+git clone https://github.com/bitcoin/bitcoin.git
+```
+
+
+## Triplets and Presets
+
+The Bitcoin Core project supports the following vcpkg triplets:
+- `x64-windows` (both CRT and library linkage is dynamic)
+- `x64-windows-static` (both CRT and library linkage is static)
+
+To facilitate build process, the Bitcoin Core project provides presets, which are used in this guide.
+
+Available presets can be listed as follows:
+```
+cmake --list-presets
+```
+
+## Building
+
+CMake will put the resulting object files, libraries, and executables into a dedicated build directory.
+This guide assumes that all commands are executed in build directory created like that:
+```
+mkdir build
+cd build
+```
+
+In following istructions, the "Debug" configuration can be specified instead of the "Release" one.
+
+### 4. Building with Dynamic Linking
+
+```
+cmake -S .. --preset vs2022  # It might take a while if the vcpkg binary cache is unpopulated or invalidated
+cmake --build . --config Release
+ctest --build-config Release
+```
+
+### 5. Building with Static Linking
+
+```
+cmake -S .. --preset vs2022-static  # It might take a while if the vcpkg binary cache is unpopulated or invalidated
+cmake --build . --config Release
+ctest --build-config Release
+cmake --install . --config Release  # Optional
```

Comment:
Is there a reason why `install` is listed as part of the static linking build and not the dynamic linking build?